newdle/client/src/components/ParticipantGrid.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import moment from 'moment';
+import _ from 'lodash';
+import {useSelector} from 'react-redux';
+import {Icon, Table} from 'semantic-ui-react';
+import {serializeDate, toMoment} from '../util/date';
+import {
+  getNewdleDuration,
+  getNewdleParticipants,
+  getNewdleTimezone,
+  getNewdleTimeslots,
+} from '../selectors';
+import UserAvatar from './UserAvatar';
+import styles from './ParticipantGrid.module.scss';
+import PropTypes from 'prop-types';
+
+function formatMeetingTime(startTime, duration) {
+  const endTime = moment(startTime)
+    .add(duration, 'm')
+    .format('HH:mm');
+  return `${serializeDate(startTime, 'HH:mm')} - ${endTime}`;
+}
+
+function DateCell({timeslot}) {
+  const newdleTimezone = useSelector(getNewdleTimezone);
+  const duration = useSelector(getNewdleDuration);
+
+  const startTime = toMoment(timeslot, 'YYYY-MM-DDTHH:mm');
+  return (
+    <Table.HeaderCell textAlign="center">
+      <div>
+        <div>{startTime.format('D MMM')}</div>
+        <div>{formatMeetingTime(startTime, duration)}</div>
+        <div>({newdleTimezone})</div>
+      </div>
+    </Table.HeaderCell>
+  );
+}
+
+DateCell.propTypes = {
+  timeslot: PropTypes.string,
+};
+
+function ParticipantRow({participant}) {
+  const timeslots = useSelector(getNewdleTimeslots);
+  let participantName = participant.name;
+  let participantEmail = participant.email;
+  // eslint-disable-next-line react/prop-types
+  let matches = participantName.match(/\b(\w)/g);
+  let initials = matches.join('');
```

Comment:
const initials = participantName.match(/\b(\w)/g).join('');

or better:

    const initials = participantName.match(/\b(\w)/g).slice(0, 2).join('');

(otherwise you get more than 2 chars in case the full name has more than two names)